codelists/assetClass.csv

```diff
@@ -0,0 +1,3 @@
+Code,Title,Description
+debt,Debt,A contract that obliges a borrower to repay a lender in accordance with a specified maturity and under specified terms. Also known as fixed income.
```

Comment:
Does " Also known as fixed income." make sense in the context of financing?

Reply:
I don't think so. I had included it because that was the term used in the OECD infrastructure financing taxonomy. However, the taxonomy is mostly written from the perspective of investment so I've removed the reference to fixed income in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/a99c4745ecaa9814504f156cfba8ccdd0edccbff